docs/RECIPE_SCHEMA.md

```diff
@@ -0,0 +1,319 @@
+# Cellpack Recipe Schema 1.0
+## Recipe definition properties
+
+### name
+
+_Required string._
+
+Unique recipe name
+
+### version
+
+_Required string._
+
+Version of the recipe
+
+
+#### example of recipe definition:
+
+```JSON
+"recipe": {
+    "version": "1.0",
+    "name": "NM_Analysis_FigureA"
+    },
+```
+----
+
+## Options properties
+
+### cancelDialog
+
+_boolean_
+
+### hackFreepts
+
+### windowsSize
+_number_
+
+### use_gradient
+
+_Optional boolean_
+
+### placeMethod
+
+_Optional enum_. One of `"RAPID"`, `"jitter"`, `"spheresBHT"`, `"pandaBullet"`
+
+Will be use if placeMethod isn't in an ingredient setup
+
+`"RAPID"`
+
+`"jitter"`
+
+`"spheresBHT"`
+
+`"pandaBullet"`
+
+### saveResult
+_boolean_
+
+### runTimeDisplay
+_boolean_
+
+### overwritePlaceMethod
+_boolean_
+
+### innerGridMethod
+_enum_. One of `"bhtree"`, `"jordan"`, `"sdf"`, `"jordan3"`, `"pyray"`, `"floodfill"`, `"binvox"`, `"trimesh"`, `"scanline"`
+ `"bhtree"`
```

Comment:
Same here: @gj210 if you could give 1 sentence descriptions of each (and which ones should be abandoned) that would be helpful. You can just do it as a rely in this comment.

Reply:
This will require some work to get these right.  Jordan3 may be our best "fast" option for now.  I can't recall, but "sdf" may be the brute-force 100% guaranteed not to leak/incredibly slow option. "trimesh" may be the option either Kevin or Megan worked on with other Berkeley students.

Reply:
I looked through some of these comments and added them in. But some of them look like the function was just copied and nobody updated the comment, so I'm not sure if they still apply